code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -56,6 +57,36 @@ Difficulty: Medium
 	. = ..()
 	transform *= 2
 
+/mob/living/simple_animal/hostile/megafauna/legion/enrage()
+	health = 1250
+	maxHealth = 1250
+	transform /= 1.5
+	loot = list(/datum/nothing)
+	crusher_loot = list(/datum/nothing)
+	var/mob/living/simple_animal/hostile/megafauna/legion/legiontwo = new /mob/living/simple_animal/hostile/megafauna/legion(get_turf(src))
+	legiontwo.transform /= 1.5
+	legiontwo.loot = list(/datum/nothing)
+	legiontwo.crusher_loot = list(/datum/nothing)
+	legiontwo.health = 1250
+	legiontwo.maxHealth = 1250
+
+/mob/living/simple_animal/hostile/megafauna/legion/unrage()
+	. = ..()
+	for(var/mob/living/simple_animal/hostile/megafauna/legion/other in GLOB.mob_list)
+		if(other != src)
+			other.loot = initial(loot)
+			other.crusher_loot = initial(crusher_loot)
+			other.maxHealth = 2500
+			other.health = 2500
+	qdel(src) //Suprise, it's the one on lavaland that regrows to full.
+
+/mob/living/simple_animal/hostile/megafauna/legion/death(gibbed)
+	for(var/mob/living/simple_animal/hostile/megafauna/legion/other in GLOB.mob_list)
+		if(other != src)
+			other.loot = initial(loot)
+			other.crusher_loot = initial(crusher_loot)
```

Comment:
Does this mean you get duplicate loot? Not sure if this is intended

Reply:
Nope! This makes the other fauna have loot, as the loot on the current legion is nothing if enraged